crates/biome_service/src/workspace/server.tests.rs

```diff
@@ -106,3 +106,30 @@ fn store_embedded_nodes_with_current_ranges() {
     let style_node = style.node();
     assert!(style_node.text_range_with_trivia().start() > TextSize::from(0));
 }
+
+#[test]
+fn call_info_for_file() {
+    let source = include_str!("./fixture.js");
```

Comment:
Let's move `fixture.js` file to `biome_module_graph/benches/` with a bit more descriptive file name. (See also the comment above.)